alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/monitors/BoundariesDrawer.java

```diff
@@ -0,0 +1,35 @@
+package it.unibo.alchemist.boundary.monitors;
+
+import it.unibo.alchemist.boundary.wormhole.interfaces.IWormhole2D;
+import it.unibo.alchemist.model.interfaces.Environment;
+import it.unibo.alchemist.model.interfaces.Position2D;
+import it.unibo.alchemist.model.interfaces.environments.BoundariesVisitor;
+import it.unibo.alchemist.model.interfaces.environments.RectangularBoundaries;
+import org.jetbrains.annotations.NotNull;
+
+import java.awt.Graphics2D;
+import java.awt.Point;
+
+/**
+ * Draws {@link it.unibo.alchemist.model.interfaces.environments.Boundaries}.
+ * @param <P> Position type
+ */
+/* default */ class BoundariesDrawer<P extends Position2D<P>> implements BoundariesVisitor {
```

Comment:
What is this comment?

Reply:
`/* default */` is required by PMD rules, it was `/* package-private */` originally

Reply:
Which rule?

Reply:
Uh, I get it. The idea is discouraging package-private access. Take a stance here: one of `public`, `private`, or `protected` (intended in the Kotlin way, namely visible to subclasses). Package access is fragile to refactoring.

Reply:
Changed to public. I'd make it internal if it was kotlin

Reply:
Changed to public. I'd make it `internal ` if it was kotlin

Reply:
Makes sense in general, but I do not like `internal` as it implies a concept that's foreign to the language (module).